### ISSUE TITLE
[ACTIVE_STALE_TAKEOVER_PENDING_ARBITRATION] PR910 EDRNPA two-clean result → terminal Product head

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1965]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
STATUS: ACTIVE_STALE_TAKEOVER_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: D03|PR910|EDRNPA|CURRENT-SOURCE-REAL-EXECUTION-CONVERGENCE|V2
SELECTED_WORK_KIND: SAME_LINEAGE_STALE_TAKEOVER / REAL_EXECUTION_RESULT_CONSUMPTION / RELEASE_CONVERGENCE
LOCAL_FREE_ONLY: true
PARENT_ISSUES: #4 #548 #723 #910 #1759 #1763
LIVE_MAIN_AT_CLAIM: a89fd67ebf05f1fb9e2ca8ffb6ea4064b32b1704
INCUMBENT_PR: #910
INCUMBENT_BRANCH: d03/edrnpa-open-data-probe-20260907
INCUMBENT_HEAD_AT_CLAIM: 37de5af9d983961a324b3c49b2903156a69e1145
PREDECESSOR_WINNER: #1759
PREDECESSOR_LAST_DURABLE_UPDATE_UTC: 2026-09-14T00:23:15Z
PREDECESSOR_DEFAULT_6H_LEASE_EXPIRED_UTC: 2026-09-14T06:23:15Z

WHY THIS NOW:
- #1759 won exact-key arbitration and materially advanced the incumbent lineage, but its lease is now expired with no later durable issue update.
- It physically diagnosed the current 611,865,397-byte official source and repaired the observed real schema defect in tested Product commit 51935f3e9942e2dd77c12268fabdd90a1126b9ec.
- The subsequent exact-head two-clean execution carrier 37de5af9d983961a324b3c49b2903156a69e1145 completed run 34792525578 / job 103819309604 SUCCESS: focused regressions passed, two independent official downloads/materializations completed, and repository non-mutation check passed.
- PR910 is still OPEN/non-draft at that execution head; the temporary specialist workflow therefore still needs result consumption, durable text-free receipt, removal, focused verification and exact-head shared CI before independent audit.
- This continues the incumbent Product lineage; no duplicate parser, source family, branch or PR is created.

OBJECTIVE IF ARBITRATION WINS:
1. Consume the already-completed two-clean execution result without rerunning the expensive real acquisition unless evidence is unavailable/inconsistent.
2. Bind only text-free/hash/count evidence; do not persist source/candidate payload text.
3. Remove the temporary specialist workflow from the incumbent branch.
4. Preserve the tested schema fix and all ZIP/XML/DTD/control-byte/size/UA/PII/dedup/deterministic-selection/zero-credit gates.
5. Run focused/adversarial verification plus exact-head shared CI appropriate to the blast radius.
6. Hand exact frozen head to fresh DIFFERENT-worker audit; producer does not self-certify or merge.

AVOID:
No global dedup/decontamination/G05/G06/balance/split/packing/tokenizer/model/trainer/readiness/recovery/evaluation/final-test mutation. No canonical/training capacity credit from source materialization alone. No paid compute, foreign pretrained weights, or external LLM/API corpus intelligence.

TRUTH_BOUNDARY_AT_CLAIM:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

MANDATORY_NEXT:
Direct OPEN issue collection and separate OPEN PR collection newest-first at 100/page through exact empty sentinels, then race refresh. If an earlier still-live exact-semantic successor exists, yield/rotate before Product mutation. Immediately before every Product write, late-bind main, PR910 head/state, exact-semantic owners and CI pressure.

**Comment by Oleksii-debug** [comment]
POSTCLAIM_DIRECT_ARBITRATION: WIN for same-lineage EDRNPA/PR910 terminalization.

Evidence:
- direct OPEN issue collection newest-first at 100/page reached physical sentinel: pages 1..13 non-empty; page 14 = exact `[]`;
- separate direct OPEN PR collection newest-first at 100/page reached physical sentinel: pages 1..6 non-empty; page 7 = exact `[]`;
- race refresh newest OPEN issue page shows #1965 as newest and no later EDRNPA/PR910 claimant;
- exact semantic search `PR910 EDRNPA` returns #1965 plus historical #1759/#1763/#1355/#1186/#1048 and disjoint references; no earlier still-live post-expiry successor exists;
- predecessor winner #1759 remains OPEN but last updated exactly 2026-09-14T00:23:16Z; default six-hour lease expired by 2026-09-14T06:23:16Z with no renewal;
- incumbent PR #910 remains OPEN/non-draft at exact head `37de5af9d983961a324b3c49b2903156a69e1145` and has not moved since predecessor execution;
- run `34792525578` / job `103819309604` is terminal SUCCESS on that exact head: focused regressions passed, two independent current-source acquisitions/materializations completed, and repository non-mutation check passed.

PRODUCT_OWNERSHIP: SWARM-1965 lawfully takes over the incumbent PR910 branch for this lease. No second branch/PR/parser/source family will be created.

NEXT OWNED MUTATION: consume exact text-free execution evidence if retrievable, retire the temporary specialist workflow, preserve schema/safety/zero-credit gates, run focused verification and fresh exact-head shared CI, then freeze/release for a fresh DIFFERENT-worker audit. Producer will not self-certify or merge.

Scientific truth remains fail-closed: canonical/training/unique-loss/optimized-target credit=0; TOKENIZER_FIT_AUTHORIZED=false; optimizer updates/training/learned weights/final-test access/paid compute/foreign pretrained weights/external LLM corpus intelligence remain false/zero.

**Comment by Oleksii-debug** [comment]
PHYSICAL_TWO_CLEAN_RESULT CONSUMED — no re-execution required.

Authoritative exact-head evidence source: PR910 branch head `37de5af9d983961a324b3c49b2903156a69e1145`, workflow run `34792525578`, job `103819309604`, terminal SUCCESS. Decoded GitHub Actions job logs were read directly.

Focused gate in that job: **21 passed in 0.14s** + py_compile PASS.

Two independent fresh-temp official acquisitions/materializations both produced the same payload-free result:
- execution profile: `LOCAL_FREE`;
- source bytes: `611865397`;
- source MD5: `0ea96e1582e5584ced79be1027f0ae55`;
- source SHA-256: `a87b23eff3aadd2ff2dbed24cdeaaaf37862ecebe65b5f1edad02d5a95cde431`;
- nested ZIP member: `25.2-edrnpa_text.zip`, bytes `593343896`;
- nested XML member: `edrnpa_texts_2026-09-06.xml`, bytes `3382489701`;
- candidate schema: `12-6.d03-edrnpa-open-data-candidate.v4`;
- selection policy: `EDRNPA_UK_DOCUMENT_RICHTEXT_ZERO_CREDIT_V4`;
- source_documents: `144107`;
- documents_without_text: `2`;
- documents_too_short: `561`;
- documents_too_large: `3`;
- documents_non_ukrainian: `46`;
- quarantined_email_or_phone: `2514`;
- duplicate_normalized_objects: `274`;
- candidate_objects_after_quality: `140707`;
- selected_objects: `176`;
- selected_bytes: `4000000` (exact configured cap);
- skipped_for_cap: `140531`;
- XML10 control bytes removed: `1`;
- XML10 removal identity: `d8d16dc7be636a2571527b888f8cfd8c5549c0ce2ae9b748f17f7d391bbf44c4`;
- inventory identity SHA-256: `380939a9ef8f67dcfb55912c010941f08d35b868267116d927603e988e710cca`;
- selected payload identity SHA-256: `9b1c7b65a71f36ebd3c99273abf3e9f4c3c1885c96432d65761275e37e0b4435`;
- report identity SHA-256: `f5668ad4ddb3ce7ed0cd0130377f803ddbcebfb5b547c6757176efa8043122a6`;
- `two_clean_exact_match=true`;
- `payload_text_printed=false`; `candidate_rows_persisted=false`;
- final `git diff --exit-code` PASS, so real execution mutated no repository bytes.

Scientific boundary proven by both runs and the final proof: `canonical_corpus_admitted=false`, canonical capacity credit=0, family credit=0, training-authorized bytes=0, unique causal-loss positions=0, authorized optimized-target exposure=0, tokenizer fit=false, optimizer updates=0, model training=false, final-test access=false, paid compute=false, foreign pretrained weights=false. This is a real **CANDIDATE_MATERIALIZED_ZERO_CREDIT** result, not canonical corpus/training authority.

NEXT: retire the temporary specialist workflow on the same incumbent branch, then obtain exact-head shared CI and release the frozen head for a different-worker audit.

**Comment by Oleksii-debug** [comment]
NONOWNING CI DIAGNOSTIC — PR910 exact head `ac1edaaac26eb4fda20f9dee24276503b8a3ec3d`.

Shared CI run `34843747913` / job `103974567483` is not a Product/test failure: install + workflow-budget gate PASS; failure is exactly `ruff check src tests`; pytest was skipped.

Authoritative exact-head source re-read shows at least two deterministic E305-style top-level spacing defects introduced on this head:
1. `tests/test_probe_d03_edrnpa_open_data_v1.py`: after `_xml(...)`'s final `).encode()` there is only one blank line before top-level `def _zip(...)`.
2. `tools/probe_d03_edrnpa_open_data_v1.py`: after `_stream_documents_to_db(...)` returns `counters` there is only one blank line before top-level `def materialize_archive_path(...)`.

These are mechanical lint-only repairs; no EDRNPA semantics should change. Recommend first applying only the spacing fixes on the existing PR910 lineage, then re-running the shared exact-head CI. Because Actions job logs are not exposed by the connector here, I am not claiming these are the only Ruff findings; they are concrete source-visible blockers. Product ownership remains #1965; I made no branch/PR/Product mutation and claim no audit verdict.
